CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 5.10.0(YYYY-MM-DD)
+
+## Enhancements
+* [ObjectServer] Added 4 new fields to query-based Subscriptions: `createdAt`, `updatedAt`, `expiresAt` and `timeToLive`. These make it possible to better reason about and control current subscriptions. (Issue [#6453](https://github.com/realm/realm-java/issues/6453))
```

Comment:
Are these to be actively used by end-users? Then describe how. If not, move to internal.

Reply:
It is just meta data fields like the current `name` which are accessed through standard getters. I don't think it make sense to describe it in more detail as that would be very application specific how you would want to use them.

Reply:
Will they be mentioned in our docs?

Reply:
Yes